src/js/components/cycleCount/toResolveTab/ResolveStepTable.jsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+import { RiAddCircleLine } from 'react-icons/all';
+import { useSelector } from 'react-redux';
+import { Tooltip } from 'react-tippy';
+
+import DataTable from 'components/DataTable/v2/DataTable';
+import Button from 'components/form-elements/Button';
+import DateField from 'components/form-elements/v2/DateField';
+import SelectField from 'components/form-elements/v2/SelectField';
+import { DateFormat } from 'consts/timeFormat';
+import useResolveStepTable from 'hooks/cycleCount/useResolveStepTable';
+import useTranslate from 'hooks/useTranslate';
+import { formatDate } from 'utils/translation-utils';
+
+import 'components/cycleCount/cycleCount.scss';
+
+const ResolveStepTable = ({
+  id,
+  product,
+  dateCounted,
+  dateRecounted,
+  tableData,
+  tableMeta,
+  addEmptyRow,
+  removeRow,
+  assignRecountedBy,
+  setRecountedDate,
+}) => {
+  const {
+    columns,
+    defaultColumn,
+    recipients,
+  } = useResolveStepTable({
+    cycleCountId: id,
+    tableData,
+    removeRow,
+  });
+
+  const translate = useTranslate();
+
+  const {
+    formatLocalizedDate,
+  } = useSelector((state) => ({
+    formatLocalizedDate: formatDate(state.localize),
+  }));
+
+  return (
+    <div className="list-page-list-section">
+      <p className="count-step-title pt-4 pl-4">
+        {product?.productCode}
+        {' '}
+        {product?.name}
+      </p>
+      <div className="d-flex">
+        <div className="pt-3 pl-4 d-flex align-items-center">
+          <div className="d-flex date-counted-container mb-2">
+            <p className="count-step-label count-step-label-date-counted mr-2 mt-2">
+              {translate('react.cycleCount.dateCounted.label', 'Date Counted')}
+              <span className="count-step-value ml-1">
+                {formatLocalizedDate(dateCounted, DateFormat.DD_MM_YYYY)}
```

Comment:
Why we are using this date format here? Weren't we recently unifying or changing the date format to a specific one?

Reply:
it's the date format from mock

Reply:
@alannadolny You mean this one: 
![image](https://github.com/user-attachments/assets/e0a8079f-a9bd-48e1-9637-4ddfd337145d)
?

Reply:
yup

Reply:
How do you know which part is the day and which part is the month?